src/test/java/com/scottlogic/LikesPostSorterTest.java

```diff
@@ -0,0 +1,114 @@
+package com.scottlogic;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LikesPostSorterTest {
+    UserPost userPost1 = new UserPost("Joe Bloggs",
```

Comment:
A general things about naming here which is a bit more obvious with this case is it is often nice to have the names reflect information about the object. At the moment `userPostX` doesn't mean `X` likes, this means that when looking through the tests you have to keep coming back up here to make sure the orderings are correct. If instead `userPostX` had `X` likes then the output list would be really obvious it was in the correct order and you could see the impact of the rearranged input list as well.